documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -263,128 +235,104 @@ public void run() {
-upper bounds. In our case, on line 11, we start at the beginning of the Stream.
-Other configuration items, such as specifying checkpointing etc. are options
-that will be available through the ReaderGroupConfig.  But for now, we keep it
-simple.
+The developer specifies the Stream which should be the part of the Reader Group and its lower and
+upper bounds. In the sample application, we start at the beginning of the Stream as follows:
 
-The fact that a ReaderGroup can be configured to read from multiple Streams is
-kind of cool.  Imagine a situation where I have a collection of Stream of sensor
-data coming from a factory floor, each machine has its own Stream of sensor
-data.  I can build applications that use a ReaderGroup per Stream so that the
-app reasons about data from exactly one machine.  I can build other apps that
-use a ReaderGroup configured to read from all of the Streams.  In our case, on
-line 14, the ReaderGroup only reads from one Stream.
+```Java
+final ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
+                                                                .stream(Stream.of(scope, streamName))
+                                                                .build();
+```
+Other configuration items, such as specifying checkpointing etc., are options
+that will be available through the `ReaderGroupConfig`.
 
-You can call createReaderGroup() with the same parameters multiple times, it
-doesn't hurt anything, and the same ReaderGroup will be returned each time after
-it is initially created.
+The Reader Group can be configured to read from multiple Streams. For example, imagine a situation where there is a collection of Stream of sensor data coming from a factory floor, each machine has its own Stream of sensor data.  We can build applications that uses a Reader Group per Stream so that the
+application reasons about data from exactly one machine. We can build other applications that
+use a Reader Group configured to read from all of the Streams. In the sample application, the Reader Group only reads from one Stream.
 
-Note that in other cases, if the developer knows the name of the ReaderGroup to
-use and knows it has already been created, s/he can use getReaderGroup() on
-ReaderGroupManager to retrieve the ReaderGroup object by name.
+We can call `createReaderGroup()` with the same parameters multiple times and the same Reader Group will be returned each time after it is initially created.
 
-So at this point in the code, we have the Scope and Stream set up, we have the
-ReaderGroup created and now we need to create a Reader and start reading Events.
+In cases when the developer knows the name of the required existing Reader Group, they can use `getReaderGroup()` on
+`ReaderGroupManager` to retrieve the Reader Group object by name. Once, the Scope and Stream is set up, then the Reader Group can be created using the API `createReaderGroup()` and we can create a Reader and start reading Events.
```

Comment:
Please, add a blank line before this sentence and consider suggested rephrase: `Once, the Scope and...` -> At this point, we have the Scope and Stream is set up and the `ReaderGroup` object created. Next, we need to create a Reader and start reading Events.

Reply:
Accepted suggestion